SUPPORTED_APIS.md

```diff
@@ -86,7 +86,7 @@ The `status` options are:
 | `EVM` | `evm_setBlockGasLimit` | `NOT IMPLEMENTED` | Sets the Block Gas Limit of the network |
 | `EVM` | `evm_setIntervalMining` | `NOT IMPLEMENTED` | Enables (with a numeric argument greater than 0) or disables (with a numeric argument equal to 0), the automatic mining of blocks at a regular interval of milliseconds, each of which will include all pending transactions |
 | `EVM` | `evm_setNextBlockTimestamp` | `NOT IMPLEMENTED`<br />[GitHub Issue #68](https://github.com/matter-labs/era-test-node/issues/68) | Works like `evm_increaseTime`, but takes the exact timestamp that you want in the next block, and increases the time accordingly |
-| `EVM` | `evm_setTime` | `NOT IMPLEMENTED` | Sets the internal clock time to the given timestamp |
```

Comment:
I think there's actually a need for both here. Ganache only supports [evm_setTime](https://ganache.dev/#evm_setTime), while Hardhat only support [evm_setNextBlockTimestamp](https://hardhat.org/hardhat-network/docs/reference#evm_setnextblocktimestamp) ... but I _think_ they both do the same thing. Thoughts?

Reply:
We can either implement `evm_setTime` in this PR, or leave the line here in this table.

Also, it looks like this change leaves 2 rows/entries for `evm_setNextBlockTimestamp`

Reply:
> I think there's actually a need for both here. Ganache only supports [evm_setTime](https://ganache.dev/#evm_setTime), while Hardhat only support [evm_setNextBlockTimestamp](https://hardhat.org/hardhat-network/docs/reference#evm_setnextblocktimestamp) ... but I _think_ they both do the same thing. Thoughts?

Yeah they do exactly the same thing with the following differences:
* `setNextBlockTimestamp` allows only setting a timestamp forward in future, while `setTime` supports setting it backwards.
* `setNextBlockTimestamp` returns the newly set timestamp in response, whereas `setTime` returns the difference between the current and the new timestamp.

AFAIK `setTime` can be seen as a superset of `setNextBlockTimestamp` and from most examples out there no one seems to be using the return value to do anything meaningful.

I'm fine with supporting both, though it must be explicitly mentioned that these two achieve the same result and are mostly arising out of api differences between hardhat and ganache - though it does tend to make our API interface a bit messy.